rfc/spying-and-tracing.md

```diff
@@ -0,0 +1,24 @@
+# Spying and Tracing
+
+This feature adds the ability to expose the inner workings of MobX. It will report over console logs or DDT about the various activities happening inside MobX. This includes:
+
+- When observables are read and written
+- When actions are invoked
+- When reactions are triggered
+
+## Benefits
```

Comment:
I think this section could use more flavor. Maybe problems that we'd like the feature to solve for people? Use cases? More detail can help shape the feature and ensure that we're solving real user needs.

Reply:
Agree. Will add more details here and how this feature helps

Reply:
I'll help. I have a few things in mind, but they're still brewing. :)

Reply:
Pushed a few changes. Let me know if this is along those lines :-)